frontend/src/views/Camps.vue

```diff
@@ -134,6 +156,9 @@ export default {
         (c) => !c.periods().items.some((p) => dayjs(p.end).endOf('day').isAfter(dayjs()))
       )
     },
+    invitations() {
+      return this.api.get().personalInvitations().items
```

Comment:
don't you need a null check here?

Reply:
No, hal-json-vuex would never return null intermediately here.
`this.api` is always defined in our Vue app, it provides access to hal-json-vuex
`this.api.get()` is a special hal-json-vuex call which returns the data from the API root endpoint (list of all collections)
`this.api.get().personalInvitations()` returns a hal-json-vuex proxy which will never resolve to null unless we rename the personal_invitations entry in the API root endpoint
`.items` is a special property for hal-json-vuex proxies, in the sense that it always returns an array, just an empty one as long as we don't have any data